rules/config

```diff
@@ -152,6 +152,9 @@ INCLUDE_DHCP_RELAY = y
 # INCLUDE_DHCP_SERVER - build and install dhcp-server package
 INCLUDE_DHCP_SERVER ?= n
 
+# INCLUDE_BMP - build and install sonic-bmp package
```

Comment:
>build [](http://example.com/codeflow?start=16&length=5)

Build time, we should "include".
Runtime, you can decide by default "enable" or not.

Reply:
Yes, I will turn this on as last step after bmp/bmpcfgd PR are merged.